src/app/workspace/results/results.component.ts

```diff
@@ -37,6 +40,11 @@ export class ResultsComponent implements OnInit {
     }
 
     openResource(id: string) {
-        this._router.navigate(['/resource/' + encodeURIComponent(id)]);
+        this.resourceIri = id;
+    }
+
+    // open project in new tab
+    openProject(project: ReadProject){
+        window.open('/project/' + project.shortcode, "_blank");
     }
```

Comment:
Why do we have to open a project in new tab?

Reply:
It was the original functionality. Do we want to change it?

Reply:
> > @kilchenmann Also I did not remove the "app-resource" component. As we are not using it anymore, should I remove it completely?
> 
> No, no we will still use it. It will be shown in case of an ark-url redirection.

Ok.

Reply:
> It was the original functionality. Do we want to change it?

Ok. No, in this case we can keep it as it is. I was wondering because it was highlighted green like new code.